dashboard/static/styles/main.styl

```diff
@@ -26,6 +26,11 @@ section {
     background-position: top center;
 }
 
+.top-bar-title h1 {
+    font-size: 1rem;
+    font-weight: normal;
```

Comment:
Huh, you're right. I must have been checking it without saving or something. Good catch. Less code == Less problems.